src/Codeception/Module/Doctrine.php

```diff
@@ -689,7 +689,7 @@ private function extractPrimaryKey(object $instance)
      *
      * This method requires [`doctrine/data-fixtures`](https://github.com/doctrine/data-fixtures) to be installed.
      *
-     * @param class-string<FixtureInterface>|class-string<FixtureInterface>[]|list<FixtureInterface> $fixtures
+     * @param class-string<\Doctrine\Common\DataFixtures\FixtureInterface>|class-string<\Doctrine\Common\DataFixtures\FixtureInterface>[]|list<\Doctrine\Common\DataFixtures\FixtureInterface> $fixtures
```

Comment:
@ThomasLandauer Are you sure that the FQCN is necessary?

IMO the [existing `use` statement](https://github.com/Codeception/module-doctrine/blob/6a848756d6d83ba96e510aaa2cf6c9f3eda482bd/src/Codeception/Module/Doctrine.php#L20) should be sufficient.